daprdocs/content/en/java-sdk-docs/java-client/_index.md

```diff
@@ -17,6 +17,44 @@ If you haven't already, [try out one of the quickstarts]({{< ref quickstarts >}}
 
 [Complete initial setup and import the Java SDK into your project]({{< ref java >}})
 
+## Initialising the client
+You can initialise a Dapr client as so:
+
+When you initialise the client without any parameters it will use the default endpoint values for a Dapr 
+sidecar instance (`127.0.0.1:50001`).
+
+```java
+DaprClient client = new DaprClientBuilder().build()
+```
+
+
+#### Environment variables:  
+
+##### Dapr Sidecar Endpoints
+You can use the standardised `DAPR_GRPC_ENDPOINT` environment variable to
+specify the gRPC endpoint. When this variable is set, the client can be initialised 
+without any arguments:
+
+```bash
+export DAPR_GRPC_ENDPOINT="mydomain:50051?tls=true"
```

Comment:
Just a small nit: `Initialising` -> `Initializing` && `standardising` -> `standardizing`. I'm not 100% confident I see the parsing for the `tls=true` in our codebase. I might've just missed it though. Once this is confirmed, then I think the PR is good to go. Thank you for documenting this 🎉

Reply:
we may be missing a conditional, like: 
```
      if (uri.getQuery() != null && uri.getQuery().contains("tls=true")) {
        insecure = false;
      }
```

Reply:
Thanks Cassie! I realised I had some copy/paste  leftovers from the python-sdk. I fixed that now.
The `?tls` query parameter was added in [this proposal](https://github.com/dapr/proposals/pull/40).
I removed the example for now, so we're not showing the grpc endpoint with "https", and we can come back to this and add it, once the support for tls is added.